### DISCUSSION TITLE
Add Icon to Title Card

### DISCUSSION THREAD

**Discussion by Does-This-Work** [UI-Lovelace-Minimalist/UI#1193] [Q&A] [answered]
Hi,
can someone point me in the right direction on how to add an icon to a title card?
What I am trying to do is add a icon right next to bedroom.
![grafik](https://user-images.githubusercontent.com/102190830/222594328-2081abd8-eb83-48c3-be62-6dd73d4f2eff.png)
Those are my current tries but I can't get it to work.
```
  - type: 'custom:button-card'
    template: card_title
    name: Bedroom mdi:bed-king-outline
    show_icon: true
```
```
  - type: 'custom:button-card'
    template: card_title
    name: Bedroom 
    show_icon: mdi:bed-king-outline
```

**Comment by basbruss** [comment, marked as answer]
It's a title card and not really designed to work with material design icons.

You can use emoticons in the name or label field if you want to.

**Reply by Does-This-Work** [reply]
Thank you! I just left out the icon and decided the color should be enough.
